doc/contributing/release_notes.rst

```diff
@@ -0,0 +1,13 @@
+How to write release notes
+==========================
+
+Release notes are generated by a script from individual changelog strings.
+Below are some best practices to make these changelog strings consistent, neat, and human-oriented.
+
+*   Use the past tense to describe changed or fixed behavior.
+    Example: 'Fixed a false positive panic when yielding in debug hook (gh-5649)'.
+*   Use the present tense to describe new behavior.
+    Example: 'Data changes in read-only mode are now forbidden (gh-5231)'.
+*   Start with a capital letter, end with a period. 
```

Comment:
Cited from the issue:

> Imperative mood is used when we start to do X: say, 'Deploy packages for Fedora 32 (gh-4966)'.

Is it forbidden? If so, how to write this change description? If not, please, include.

Reply:
I'm not sure I understand why you need the imperative mood in the release notes. This document is not telling the users what to do but describing what has been done.
'Deploy packages for Fedora 32 (gh-4966)'—doesn't it mean that "packages have been deployed"?

Reply:
It means that a necessary machinery is integrated into the repository, so the release will be deployed for given OS, when it'll be released. (Also it implicitly means that we start to test commits against this OS to don't break anything here.) The outcome for a user: download instructions on the website will work for given OS after the release. If you don't like this wording, can you propose one that satisfies you? So I would able to compare and either agree or give arguments against.

Reply:
My proposition (if I understood you right):
(Tarantool) packages will be available for Fedora 32 on release.

"Deploy" can be understood both as an imperative and as present tense, which is confusing.

Reply:
So it'll look so in the release notes:

> * Packages will be available for Fedora 32 on release (gh-4966).

Release notes are published at release, so packages should be already available. So 'will be' and 'on release' looks strange in this context for me.

If we're going this way, maybe someting like this?

> * Fedora 32 is now supported. Per-commit testing and deployment of prereleases and releases to the YUM repository is added (gh-4966).

Whether this variant looks clear for you?

----

Is seems, we should collect examples for such situations: where we start to discuss how to better reflect a change. I propose to add an example for a new supported distribution after this discussion.

Reply:
Regarding my first example: I meant the Fedora release, not the Tarantool release.

Regarding the second example:

- I would start the second sentence with "Added" as per other rules.
- I still don't understand why we're explaining to the reader what we are currently doing, or started doing. Release notes are about what has already been done, finished, included in the release. Could you probably explain it in Russian and/or in an oral conversation?

Here is another attempt—does the following make sense?
"Added packages for the upcoming Fedora 32 release."

Reply:
We usually add packages, when a distribution is already released.

> I would start the second sentence with "Added" as per other rules.

Ok.

> I still don't understand why we're explaining to the reader what we are currently doing, or started doing.

We don't, as I see. Ok, let's discuss voicely.

The result:

> * Fedora 32 is supported now. Added per-commit testing and deployment of prereleases and releases to the YUM repository (gh-4966).

Reply:
After the voice discussion:

> * Fedora 32 is supported now. Added per-commit testing, updated YUM repositories (gh-4966).

Reply:
I ask you again to respond to comments, when they're fixed.

This one is fixed now, I'm closing the discussion.